demo/full-featured.md

```diff
@@ -9,6 +9,8 @@ keywords: example demo custom wysiwyg full-power full-featured plugins non-premi
 
 ## Full Featured: Including Premium Plugins
 
+> **Important**: This demonstration page is for {{site.productname}} {{site.productmajorversion}}. {{site.productname}} {{site.productmajorversion}} is still a supported version but it is not the current release. The full-featured demonstration page for {{site.productname}} 6 is [available]({{site.baseurl}}/tinymce/6/full-featured-premium-demo/) in the {{site.productname}} 6 [documentation]({{site.baseurl}}/tinymce/6/).
+
```

Comment:
```suggestion
```
See https://github.com/tinymce/tinymce-docs/pull/2489#discussion_r944023506, this was not what had agreed to and just causes confusion by having it on a single page.